proto/pocket/migration/event.proto

```diff
@@ -108,4 +108,20 @@ message EventMorseSupplierClaimed {
   // Supplier staked as a result of the claim
   // - Mirrors Morse supplier stake
   shared.Supplier supplier = 5 [(gogoproto.jsontag) = "supplier"];
-}
\ No newline at end of file
+}
+
+// EventMorseAccountRecovered
+// - Emitted when a Morse account is recovered on-chain
+message EventMorseAccountRecovered {
+  // The session end height (on Shannon) in which the recovery was committed (i.e. claimed).
+  int64 session_end_height = 1 [(gogoproto.jsontag) = "session_end_height"];
+
+  // The total balance which was recovered.
+  cosmos.base.v1beta1.Coin recovered_balance = 2 [(gogoproto.jsontag) = "recovered_balance", (gogoproto.nullable) = false];
+
+  // The bech32-encoded address of the Shannon account to which the recovered balance will be minted.
+  string shannon_dest_address = 3 [(cosmos_proto.scalar) = "cosmos.AddressString", (gogoproto.jsontag) = "shannon_dest_address"];
+
+  // The hex-encoded address of the Morse account whose balance and stakes have been recovered.
```

Comment:
```suggestion
  // The hex-encoded address of the Morse account whose balance and stakes have been recovered.
  // If funds are being recovered from an unreachable address, this may be an invalid Morse address.
```

Reply:
Haven't gone through the rest of the PR, but please confirm if this is accurate or not (@bryanchriswhite @red-0ne)

Reply:
@Olshansk, I updated the comment, could you PTAL?